.changes/next-release/Feature-a468d30b-6593-44ea-9ab3-2d4bb93ec1e8.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Add support for configuring max amount of CloudWatch Logs entries pulled per service call"
```

Comment:
```suggestion
	"description": "Users can configure the CloudWatch Logs page size (number of entries pulled per service call)"
```

Reply:
IIRC we've had discussions in the past about "Users can X" in changelog entries; as far as I can tell we don't have that phrase in our codebase at this time.

Reply:
My suggestion is to avoid referring to users in third person in changelog entries, and to state the change as plain and concise as possible. Both of these are style remarks in the interest of producing consistent changelog entries.

My suggestion is...
The amount of CloudWatch Logs entries retrieved per request is now configurable.

Reply:
I was trying to avoid the passive voice, which is more difficult to follow as a reader. Perhaps "You" instead of "Users"